network/node4/genesis.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
if these are static hardcoded config files i'd rather have them as part of tendermint dev image. or atleast avoid committing multiple times to avoid cluter

Reply:
Think of each node as a totally separate and isolated entity. Each node could be in different galaxies. They won't share these files.